README.md

```diff
@@ -1 +1,144 @@
+
+- `initial_draft`
+- `technical_review`
+- `instructional_review`
+- `editorial_review`
+- `final_draft`
+
+Each branch is structured by the author, meaning you must create a folder with your name in each branch, if it doesn't exist yet. You can add additional subfolder for organization, but you don't have to.
+
+You must create your own new working branch at every step and work on your draft in that branch. Your working branch is always a copy of the latest review step branch that your draft was accepted into.
+
+To progress to the next step in the review process, create a pull request using your working branch and open it against one of the five review step branches that represents the review step that you are ready for.
+
+Your working branch can always be a copy of the latest review step branch that your article was merged into.
+
+For example, if you do not have a draft yet, you can create a branch named `myarticle1`. In that branch, write your article or upload it as a `.docx` file. Once you are ready for you draft to appear in the `initial_draft` branch, create a pull request with `myarticle1` as your source and `initial_draft` as your destination.
+
+The review occurs in the pull request. The reviewer can add comments, make suggestions, and directly approve the pull request.
+
+After your pull request is approved and merged, your draft appears in `initial_draft` and you can delete the `myarticle1` branch.
+
+Now, you can proceed to the next step by creating a copy of `initial_draft` named `myarticle2` and making any changes you need to it. When you are ready for technical review, follow the same steps as previously.
+
+After approval, you can merge the pull request and your tech reviewed article will appear in the `technical_review` branch. Repeat the process the remaining review steps.
+
+You can find a detailed breakdown in the following topics:
+
+# Markdown workflow
+
+See the following topics to learn how to upload your article and get it reviewed.
+
+## Initial draft
+
+### Prerequisite
+
+- You must have been onboarded and added to the project as a contributor
+
+### Workflow
+
+An initial draft is required before you can receive review. Complete the following steps to get your new article into the `initial_draft` branch:
+
+1. Create a new working branch. Use `initial_draft` as your **Base**. Name it using the following temple: `<your_initials>_new`
+
+2. In your working branch, create a `<article_name>.md` file in `/article/<author_name>`
+
+3. Write your article in Markdown in the `<article_name>.md` file and commit your changes.
+
+4. Create a pull request. Use your working branch `<your_initials>_new` as the **compare** and use `initial_draft` as the the **base**.
+
+5. Wait for a reviewer to approve your pull request. Address any comments as needed.
+
+6. After you receive approval, merge your pull request and delete your working branch.
```

Comment:
I tried to reach a middle ground between keeping it short and simple and detailed enough for new GitHub users to understand. Once the draft is ready, we will need someone to test this to see if it's easy to understand.